tensorflow/core/distributed_runtime/rpc/grpc_server_lib.h

```diff
@@ -36,6 +36,13 @@ namespace tensorflow {
 class GrpcWorker;
 class Master;
 
+typedef std::function<RendezvousMgrInterface*(const WorkerEnv*, 
+          const std::string& worker_name, WorkerCacheInterface* worker_cache)> 
+        RendezvousMgrCreationFunction;
+
+typedef std::function<void(const WorkerEnv*, ::grpc::ServerBuilder*)> 
```

Comment:
Could you document what this is supposed to do?  I think it's an initialization for a derived class.  So maybe call it ServiceInitFunction?